src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php

```diff
@@ -324,6 +324,9 @@ private function convertParameters(array $parameters, string $type, \DOMElement
                         }
                     }
                 }
+                if (!$excludeSelf = $tag->excludeSelf()) {
```

Comment:
no need for the $excludeSelf variable (same in the yaml dumper)

Reply:
Good point, done https://github.com/symfony/symfony/compare/698361a81af67199510ec429e630ac16641d41f4..09c758142d57c6fddffe9813b18aaaa52e054387